tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/tutorial.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Golden testing in Haskell with hspec-golden
+published: 2019-08-10
+tags: haskell, golden testing, snapshot testing, expect testing
+libraries: hspec-golen-0.1.0.0
+language: haskell
+author: Adrian Vera
+author-name: Adrian Vera
+twitter-profile: adrian_med
+github-profile: medAdrian
+description: In this tutorial we will implement golden tests in Haskell using hspec-golden a super set of the known test framework hspec. A golden test is an Input/Output action that records the result of a test into a file known as the "Golden File".
```

Comment:
What does it mean for hspec-golden to be a superset of hspec?

Reply:
To me it means it’s a layer around hpsec with more methods and that makes you follow a certain way of development that you don’t have in vanilla hspec (golden testing in this case), but if you could think on a better way of giving an explanation please provide me some guides on how could we do this, thanks. :)

Reply:
Thanks, that makes it clearer. It's the word *superset* what confuses me.